wallet/httpclient.go

```diff
@@ -6,9 +6,11 @@ import (
 	"crypto/x509"
 	"encoding/json"
 	"fmt"
+	"github.com/Qitmeer/qitmeer-wallet/globalvariable"
```

Comment:
It's not a good `golang` package name. see https://blog.golang.org/package-names